### PR TITLE
Navigation drawer MVP implementation

### PR DESCRIPTION
**Pull request by blazsolar**

### CODE REVIEW

**Comment by xaiki** [comment]
@aldoborrero care to review ?

**Comment by aldoborrero** [comment]
Yup, doing it and testing

File: mobile/src/main/java/butter/droid/ui/main/MainActivity.java [hunk 1]
```diff
@@ -82,24 +86,31 @@
     @Inject ProviderManager providerManager;
 
     @BindView(R.id.toolbar) Toolbar mToolbar;
```
Comment (aldoborrero) [inline mobile/src/main/java/butter/droid/ui/main/MainActivity.java]: Change the variable name here also removing the m suffix (as the others has been changed)

File: mobile/src/main/java/butter/droid/ui/main/navigation/NavigationDrawerFragment.java [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+                .build()
+                .inject(this);
+
+        int savedPosition = 0;
+        if (savedInstanceState != null) {
+            savedPosition = savedInstanceState.getInt(STATE_SELECTED_POSITION);
+        }
+
+        presenter.onCreate(savedPosition);
+
```
Comment (aldoborrero) [inline mobile/src/main/java/butter/droid/ui/main/navigation/NavigationDrawerFragment.java]: Let's remove this unnecessary empty space

**Pull request merged**